assets/scss/main.scss

```diff
@@ -141,22 +137,37 @@ body {
         @media (max-width: 1000px) {
             left: 0;
             right: 0;
-            top: 0;
+            top: 76px;
             bottom: auto;
             align-items: center;
-            padding: 0 10px;
+            padding: 0 40px;
             width: auto;
             display: flex;
             flex-wrap: nowrap;
             height: 70px;
-            padding-right: 50px;
+            box-shadow: none;
+
+            .overlay {
+              display: block;
+            }
 
             #menu-toggle { display: block; }
             #logo {
-                flex: 1;
-                border-left: 1px solid $grey-bg;
-                margin-left: 10px;
-                padding-left: 10px;
+                font-size: 16px;
+                margin: 0;
+                padding: 0;
+                width: fit-content;
+                padding-right: 20px;
+                &::after {
+                  font-family: 'Font Awesome 5 Free';
+                  font-weight: 900;
+                  content: "";
```

Comment:
This should be unicode. Also, instead of showing caret, show ['bars'](https://fontawesome.com/icons/bars?style=solid).